_rules/SC2-5-3-name-from-content-included-in-accessible-name.md

```diff
@@ -0,0 +1,186 @@
+---
+name: Name from content included in accessible name
+description: |
+  Interactive elements labelled through their content must have their visible label as part of their accessible name.
+
+success_criterion:
+- 2.5.3 # Label in Name
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML or SVG element that:
+* has a [semantic role](#semantic-role) that is a [widget role](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [supports name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), and 
+* has [descendants](https://www.w3.org/TR/dom41/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [visible](#visible) [text nodes](https://www.w3.org/TR/dom/#text), and
+* has an [accessible name](#accessible-name), even if the name is the empty string ("").
+
+**Note**: [widget roles](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [support name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent) are: `button`, `checkbox`, `gridcell`, `link`, `menuitem`, `menuitemcheckbox`, `menuitemradio`, `option`, `radio`, `searchbox`, `switch`, `tab`, `treeitem`.
+
+**Note:** If a text node only has [whitespace](#whitespace), it is not [visible](#visible).
+
+### Expectation
+
+The [visible](#visible) [text nodes](https://www.w3.org/TR/dom/#text) that are [descendants](https://www.w3.org/TR/dom41/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) of the target element are [included](#included-characters) in their entirety within the [accessible name](#accessible-name) of the element, unless the [text nodes](https://www.w3.org/TR/dom/#text) does not express anything in [human language](https://www.w3.org/TR/WCAG21/#dfn-human-language-s) and therefore does not live up to the [WCAG definition of text](https://www.w3.org/TR/WCAG21/#dfn-text).
```

Comment:
editorial: does > do (twice)

If the expectation is written using "unless" I fear this could be interpreted as applicability (at least that was my first interpretation). I would suggest replacing "unless" with "or". For me it is easier to interpret (the text nodes are included or they don't express anything in human language)